archive.sh

```diff
@@ -11,6 +11,29 @@ VERSION_RAW="$(sed -n 's|<version>\(.*\)</version>|\1|p' ./FinSearchUnified/plug
 VERSION="$(echo -e "${VERSION_RAW}" | tr -d '[:space:]')"
 echo "Version: ${VERSION}"
 
+STASH=$(git stash)
+
+echo ${STASH}
+
+git fetch --all --tags --prune
+
+TAG=$(git tag -l "v${VERSION}")
+
+# Get current working branch
+BRANCH=$(git branch | grep \* | cut -d ' ' -f2)
+
+# If tag is available we proceed with the checkout and zip commands
+# else exit with code 1
+if [[ -z "${TAG}" ]]
+then
+    echo "[ERROR]: Tag ${TAG} not found"
+    exit 1
+else
+    echo "Checking out ${TAG} ... "
```

Comment:
Since the branch info was dropped from this message, the else part can be deleted completely since `git checkout` will already print a similar message.